README.md

```diff
@@ -5,6 +5,8 @@ Set up with `npm install`
 
 `npm start` runs the client locally at localhost:8080 (or you can use /webpack_dev_server/ if you prefer)
 
-`./runserver` runs the server locally at localhost:9090
+`./runserver` runs the server locally at localhost:8080
```

Comment:
./runserver, the node socket server, does run on port 9090. but when you open browser windows that should be 8080 because that talks to the webpack dev server a.k.a. `npm start`.